Model/JWTRequest.php

```diff
@@ -72,7 +72,12 @@ private function buildAuthHeader($method, $restUrl)
 
     private function buildURL($restUrl)
     {
-        return $this->tenant->getBaseUrl().$restUrl;
+        // Jira return absolute self links, so its more easy to work with get with absolute urls in such cases  
```

Comment:
Its for the Jira REST responses. "Self" array items have absolute urls, not relative